src/Resources/contao/config/default.php

```diff
@@ -159,4 +159,3 @@
 $GLOBALS['TL_CONFIG']['proxyServerIps']       = '';
 $GLOBALS['TL_CONFIG']['sslProxyDomain']       = '';
 $GLOBALS['TL_CONFIG']['maintenanceMode']      = true;
-$GLOBALS['TL_CONFIG']['errorReporting']       = E_ALL & ~E_NOTICE & ~E_STRICT & ~E_DEPRECATED;
```

Comment:
Not necessarily. If we are using it in the new code, it should be defined in the `parameters.yml` file.